src/client/client.ts

```diff
@@ -188,13 +189,15 @@ export type QueueRequest = {
 
 export class Client {
   public http: Requester;
+  public chat: Chat;
```

Comment:
Let's keep the convention and move this a function. So users can invoke like:

`new Client().chat().prompt()`

Similar to,

`new Client().dlq().listMessages()`